app/services/request_password_reset.rb

```diff
@@ -26,7 +26,7 @@ def send_reset_password_instructions
     if throttle.throttled?
       analytics.throttler_rate_limit_triggered(throttle_type: :reset_password_email)
       irs_attempts_api_tracker.forgot_password_email_rate_limited(email: email)
-    else
+    elsif !user.suspended?
```

Comment:
My expectation is that we would send an email, but that we're still working on the email copy. Can we stub out a branch for the email or put some temporary copy in one?